### DISCUSSION TITLE
Pemenang AWS POP Quiz 01 - Periode 22 Mei 2023 - 26 Mei 2023

### DISCUSSION THREAD

**Discussion by rioastamal** [awsugid/forum#9] [AWS Pop Quiz]
## Pemenang AWS Quiz 01

Selamat kepada **Rosyi Ardilab** yang menjadi pemenang AWS POP Quiz 01. AWS Credit senilai **$25** akan dikirim oleh panitia ke alamat email pemenang.

## Jawaban dari Quiz 01 (Tema: EC2)

1. Berikut ini yang bukan merupakan metode untuk terhubung ke Amazon EC2 Linux adalah?

- [ ] SSH client
- [ ] EC2 Instance Connect
- [ ] AWS Systems Manager Session Manager
- [x]  Windows Remote Desktop

2. Perhatikan detail dari EC2 Instance yang telah berhasil dibuat di bawah ini. Berdasarkan aturan penamaan instance type dari Amazon EC2, angka 2 pada t2.micro menandakan?

<img width="1393" alt="AWS POP Quiz 01 - Soal 02" src="https://github.com/awsugid/forum/assets/469847/66eef046-0010-4659-93d7-2864516c07ba">

- [ ] Instance family
- [x] Instance generation
- [ ] Attribute
- [ ] Instance size

3. Gambar di bawah ini merupakan security group/firewall untuk sebuah EC2 instance. Pernyataan yang benar berdasarkan gambar adalah:

<img width="1412" alt="AWS POP Quiz 01 - Soal 03" src="https://github.com/awsugid/forum/assets/469847/a157f154-b88e-4d9f-8e11-23a19bbc7f6a">

- [ ] Pengguna dapat terhubung ke EC2 dengan SSH pada port 22 dan dari IP mana saja
- [x] Pengguna dapat terhubung ke EC2 dengan SSH pada port 22 dan dari IP 114.142.168.6/32
- [ ] Pengguna dapat terhubung ke EC2 dengan SSH pada port 22 selain dari IP 114.142.168.6/32
- [ ] Pengguna tidak dapat terhubung ke EC2 dengan SSH pada port 22 dari IP 114.142.168.6/32

## Tanya jawab

Jika ada pertanyaan seputar soal-soal diatas bisa langsung posting di thread ini. Pembahasan jawaban yang lebih lengkap dapat dilihat di channel YouTube [AWS User Group Indonesia](https://www.youtube.com/watch?v=_3HPBD9Mr0s) atau [Indonesia Belajar](https://www.youtube.com/watch?v=K292Fw6Prmc)

**Comment by misskecupbung** [comment]
Yeay! Selamat mas **Rosyi Ardilab** 🎉
